CONTRIBUTING.md

```diff
@@ -59,7 +59,7 @@ The file structure is similar to this:
   title: Agent fields
   group: 2
   description: >
-    The agent fields contain all the data about the agent/client/shipper that collected/generated the events.
+    The agent fields contain the data about the software entity, if any, that detects or observes events on a host, or takes measurements on a host. Examples include beats. Agents do not run on devices. ECS agent.* fields shall be populated with details of the agent running on the host where the event happened or the measurement was taken.
```

Comment:
I'm running Filebeat and Metricbeat on my Ubiquity EdgeMax to collect logs and system metrics. In this case my agent runs on a device I think?

Could add `detects, collects or observes` events because I think collection is an important part of most beats.

Reply:
@ruflin Thanks! yes, there are a few cases where an agent may run on devices. I will fix this language to allow for that.

Reply:
... and yes, adding "collects" makes good sense.  Will do.